ros/src/waypoint_updater/test_waypoint_updater.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+# ## in order to run tests first time follow the procedure below (despite how weird it is)
+
+# catkin_make
+# source devel/setup.bash
+# touch src/waypoint_updater/CMakeLists.txt
+# catkin_make run_tests
+
+# ## touch is used to trigger CMake execution once again after the first build and is required due to some bug in Catkin
+# https://github.com/ros/catkin/issues/974
+# Every next run of tests can be triggere by
+# catkin_make run_tests
```

Comment:
although I followed the procedure above, running the tests yields in errors "time is not initialized. Have you called init_node()?"

Reply:
Can you please provide the set of commands you used and the exact error you got. Maybe just copy all output of the console you got when trying to run the tests

Reply:
Found the problem. It was happening on fresh OS run